partitioner-config-service-api/src/main/proto/org/hypertrace/partitioner/config/service/v1/partitioner_config_service.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package org.hypertrace.partitioner.config.service.v1;
+
+option java_multiple_files = true;
+
+
+service PartitionerConfigService {
+
+  rpc GetPartitionerProfile(GetPartitionerProfileRequest) returns (GetPartitionerProfileResponse) {}
+
+  rpc PutPartitionerConfig(PutPartitionerConfigRequest) returns (PutPartitionerConfigResponse) {}
+
+  rpc GetAllPartitionerConfig(GetAllPartitionerConfigRequest) returns (GetAllPartitionerConfigResponse) {}
+
+  rpc DeletePartitionerConfig(DeletePartitionerConfigRequest) returns (DeletePartitionerConfigResponse) {}
+
+}
+
+message GetPartitionerProfileRequest {
+  string profile = 1; // not optional
+}
+
+message GetPartitionerProfileResponse {
+  PartitionerProfile profile = 1;
+}
+
+message PutPartitionerConfigRequest {
+  PartitionerProfile profile = 1;
+}
```

Comment:
Up to you, but we could trivially allow writing multiple profiles here. Should fix the Config -> Profile bit in name either way though.

```suggestion
message PutPartitionerProfilesRequest {
  repeated PartitionerProfile profiles = 1;
}
```

Reply:
done